storage/persistent_storage/persistent_storage_local/ways-to-provision-local-storage.adoc

```diff
@@ -0,0 +1,64 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ways-to-provision-local-storage"]
+= Local storage overview
+include::_attributes/common-attributes.adoc[]
+:context: ways-to-provision-local-storage
+
+toc::[]
+
+You can use any of the following solutions to provision local storage:
+
+* HostPath Provisioner (HPP)
+* Local Storage Operator (LSO)
+* Logical Volume Manager Storage (LVMS)
+
+[WARNING]
+====
+These solutions only support provisioning node-local storage. The workloads are bound to the nodes that provide the storage. If the node becomes unavailable, so does the workload. To maintain workload availability despite node failures, you must ensure storage data replication through active or passive replication mechanisms.
```

Comment:
The _only_ is supposed to be as close to the term it modifies as possible 

```suggestion
These solutions support provisioning only node-local storage. The workloads are bound to the nodes that provide the storage. If the node becomes unavailable, so does the workload. To maintain workload availability despite node failures, you must ensure storage data replication through active or passive replication mechanisms.
```

Reply:
```suggestion
These solutions only support provisioning node-local storage. The workloads are bound to the nodes that provide the storage. If the node becomes unavailable, the workload is also unavailable. To maintain workload availability despite node failures, you must ensure storage data replication through active or passive replication mechanisms.
```

Reply:
Updated as per the suggestion.